webapp/app/dashboard/views/metrics/metrics.controller.js

```diff
@@ -33,6 +37,22 @@
       })
     }
 
+    function getNullReferences() {
+      MetricService.getReferencesCodesNull()
+      .then(function(referencesNull){
+        vm.nullReferences = referencesNull.data;
+        console.log(vm.nullReferences);
+      })
+    }
+
+    function getNotNullReferences() {
+      MetricService.getReferencesCodesNotNull()
```

Comment:
Change the route, instead of `referencesCodesNotNull` use something more descriptive like: `redeemedCodes`